### DISCUSSION TITLE
strongswan scenario on rapsberry

### DISCUSSION THREAD

**Discussion by yft22** [strongswan/strongswan#786] [Q&A]
Hello Everyone,

I try to set up one of the test performed in continuous integration: https://www.strongswan.org/testing/testresults/openssl-ikev2/ecdsa-certs/index.html, I want to do it with raspberry boards (one for moon one for carol and one for David for example) I have set up strongswan on my raspberry (strongswan-swanct  strongswan and some other packages installed)  but when I look into the console.log of the test: https://www.strongswan.org/testing/testresults/openssl-ikev2/ecdsa-certs/console.log the dameon should be started on each pc (systemctl start strongswan) but I do not find it? I try to compile strongswan from source (openssl too), but that's the same, it seems that the strongswan daemon is not generated.

I guess it will be even harder to reproduce this scenario with windows PCs (I have compiled charon-svc.exe and swanctl.exe binaries)

**Comment by tobiasbrunner** [comment]
If you want the systemd-integrated daemon, you have to install `charon-systemd`. Make sure you uninstall your custom build before you use the distribution packages again (use `make uninstall` in the build directory).

**Reply by yft22** [reply]
Ok thanks,

So to reproduce the scenario I launch the charon-systemd  service (on Moon and carol for example) and then swanctl --initiate --child home?

I have updated with the scenario config file:
/etc/swanctl/swanctl.conf
/etc/strongswan.d/swanctl.conf

but I get CHILD_SA config home not found ...

**Reply by tobiasbrunner** [reply]
Read the daemon's log. But also read [ConfigurationExamplesNotes](https://wiki.strongswan.org/projects/strongswan/wiki/ConfigurationExamplesNotes).
